server/src/com/cloud/usage/UsageServiceImpl.java

```diff
@@ -224,11 +224,11 @@ public boolean generateUsageRecords(GenerateUsageRecordsCmd cmd) {
             throw new InvalidParameterValueException("Incorrect Date Range. Start date: " + startDate + " is after end date:" + endDate);
         }
         TimeZone usageTZ = getUsageTimezone();
-        Date adjustedStartDate = computeAdjustedTime(startDate, usageTZ, true);
-        Date adjustedEndDate = computeAdjustedTime(endDate, usageTZ, false);
+        Date adjustedStartDate = computeAdjustedTime(startDate, usageTZ);
+        Date adjustedEndDate = computeAdjustedTime(endDate, usageTZ);
 
         if (s_logger.isDebugEnabled()) {
-            s_logger.debug("getting usage records for account: " + accountId + ", domainId: " + domainId + ", between " + startDate + " and " + endDate +
+            s_logger.debug("getting usage records for account: " + accountId + ", domainId: " + domainId + ", between " + djustedStartDate + " and " + adjustedEndDate +
```

Comment:
Can you fix the typo here, s/djustedStartDate/adjustedStartDate/ -- this is failing the build.
You may also squash the commits, and do a git push -f with your branch on your repo.